docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+      ]
+    },
+    {
+      "metadata": {
+        "id": "8CL_lxdn8-Sv",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "When training with Keras's [Model.fit()](https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#fit), adding the [TensorBoard callback](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/callbacks/TensorBoard) will ensure logs are created and stored. Additionally, enable histogram computation every epoch with `histogram_freq=1` (this is off by default)\n",
+        "\n",
+        "Place the logs in a timestamped subdirectory to allow easy selection of different training runs."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "WAQThq539CEJ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "model = create_model()\n",
+        "model.compile(optimizer='adam',\n",
+        "              loss='sparse_categorical_crossentropy',\n",
+        "              metrics=['accuracy'])\n",
+        "\n",
+        "log_dir=\"logs/fit/\" + datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+        "tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1)\n",
+        "\n",
+        "model.fit(x=x_train, \n",
+        "          y=y_train, \n",
+        "          epochs=5, \n",
+        "          validation_data=(x_test, y_test), \n",
+        "          callbacks=[tensorboard_callback])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "asjGpmD09dRl",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Start TensorBoard through the command line or within a notebook experience. The two interfaces are generally the same. In notebooks, use the `%tensorboard` line magic. On the command line, run the same command without \"%\"."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "A4UKgTLb9fKI",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs/fit"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "MCsoUNb6YhGc",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<!-- <img class=\"tfo-display-only-on-site\" src=\"images/quickstart_model_fit.png\"/> -->"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Gi4PaRm39of2",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "A brief overview of the plugins shown (tabs in top navigation bar):\n",
+        "\n",
+        "* The **Scalars Plugin** shows how the loss and metrics change with every epoch. You can use it to also track training speed, learning rate, and other scalar values.\n",
```

Comment:
We call these *dashboards* rather than *plugins* in user-facing
documentation:
<https://github.com/tensorflow/tensorboard#the-visualizations>

(Users don’t really care whether these are pluggable components or baked
in to the core—for a long time, they _were_ baked in to the core (more
than they were now).)

Reply:
Thanks for the heads-up! Updated.